### DISCUSSION TITLE
Multi cluster notifications? How to make an alert show which cluster an alert is coming from?

### DISCUSSION THREAD

**Discussion by edvillanueva** [fluxcd/flux2#2960] [Q&A] [answered]
Hi! I've been playing around with flux notifications recently and I was following this guide https://fluxcd.io/docs/guides/notifications/ and we're able to get it running successfully. As it's sending messages to our slack channel. However, we have multiple clusters and although this is only enabled in one of our clusters, the plan is that it will be enabled through all of them. So it would be quite hard to keep track off which cluster an alert is coming from, is this something that we can configure or get help on? I've looked for guides or relevant solutions regarding this topic however I haven't seen a solution for it yet. Would really appreciate the help. Thank you so much.

**Comment by stefanprodan** [comment, marked as answer]
There is a `spec.summary` field where you can specify the cluster name and impact for alerts.

Please see the examples here: https://fluxcd.io/docs/components/notification/alert/#example

**Reply by edvillanueva** [reply]
Thank you. This indeed works! For others who may find this question. The way I implemented it is. 

spec:
  summary: "Cluster - (cluster name here)"
